pkg/plugins/runtime/k8s/metadata/annotations.go

```diff
@@ -66,6 +66,7 @@ const (
 // Annotations that are being automatically set by the Kuma Sidecar Injector.
 const (
 	KumaSidecarInjectedAnnotation                  = "kuma.io/sidecar-injected"
+	KumaSidecarUID                                 = "kuma.io/sidecar-uid"
```

Comment:
Is it for consistency or is there a legit use case to override the default UID in Kuma CP settings?

Reply:
This will allow for the CNI to read all the needed arguments from the Pod annotations in order to figure out the Transparent proxy args. Today the UID is somehow "hardcoded' in both places. We'd rather let the Mutating webhook to actually set these annotations s and the CNI will read them.

Reply:
I see, thanks for the explanation